File: DataStructures/Entry.java
public class Entry<E,V>{
    private E key;
    private V value;

    public Entry(E e, V v){
        this.key = e;
        this.value = v;
    }

    public void setKey(E key) {
        this.key = key;
    }

    public void setValue(V value) {
        this.value = value;
    }

    public E getKey() {
        return key;
    }

    public V getValue() {
        return value;
    }
}

File: DataStructures/Entry.js
var Entry = function Entry(key, value){
  this.key = key;
  this.value = value;
  this.getValue = function(){
    return this.value;
  }
  this.setValue = function(value){
    this.value = value;
  }
  this.getKey = function(){
    return this.key;
  }
  this.setKey = function(key){
    this.key = key;
  }
}
